inventory/idm-server-aws/group_vars/all.yml

```diff
@@ -0,0 +1,24 @@
+---
+pkg_update: true
+hosting_infrastructure: aws
+ansible_user: ec2-user
+
+# Name tag for the existing VPC
+vpc_name: labs
+aws_region: us-east-1
+# This should be a Gold AMI for Red Hat Linux
+ami_id: ami-0d70a070
+dns_domain: example.com
+
+instance:
```

Comment:
Currently the code (which I inherited) assumes that only one instance is being created. So indicating an array here would be misleading.  This would require much greater change to the playbooks at this point.  I do not have a problem organizing the variables under a single var - just not an array.

Reply:
@tylerauerbeck I was unable to figure out how to handle multiple instances because of the EIP associations...tricky ansible code. So I've gone with a single aws_instance, not an array. After the merge if someone want's to convert to an array and figure out the EIP business...they are welcome to.